sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs

```diff
@@ -94,16 +97,15 @@ private void BackoffConcluded()
         /// This State Machine uses delegates for the callback action and the backoff evaluation.
         /// These delegates are updated whenever a state is changed.
         /// </summary>
-        /// <param name="cancellationToken"></param>
-        private void Run(CancellationToken cancellationToken)
+        private void Run()
         {
-            while (!cancellationToken.IsCancellationRequested)
+            while (true)
```

Comment:
The loop lacks error handling. It's important to catch and properly handle any exceptions that might occur during the execution of `_callbackAction.Invoke()` to prevent the thread from exiting unexpectedly.